Samples/DataPortalPerf/DataPortalPerf/RetrieveEntities.cs

```diff
@@ -0,0 +1,12 @@
+﻿using BenchmarkDotNet.Attributes;
+using Csla;
+
+namespace DataPortalPerf
+{
+  [MemoryDiagnoser]
+  public class RetrieveEntities
+  {
+    [Benchmark(OperationsPerInvoke = 100)]
```

Comment:
I've never used `OperationsPerInvoke`, and I'm trying to figure out why you'd use it. I'm not saying it's wrong, just curious why this was done.